ddtrace/debugging/_session.py

```diff
@@ -0,0 +1,104 @@
+from dataclasses import dataclass
+import typing as t
+from weakref import WeakKeyDictionary as wkdict
+
+from ddtrace import tracer
+
+
+SessionId = str
+
+
+def _sessions_from_debug_tag(debug_tag: str) -> t.Generator["Session", None, None]:
+    for session in debug_tag.split("."):
+        ident, _, level = session.partition(":")
+        yield Session(ident=ident, level=int(level or 0))
```

Comment:
maybe we want default level to be different. level zero is same as not passing the session at all.